src/brands/kate-spade/default/font/family.json

```
@@ -13,6 +13,14 @@
         "normal": {
           "value": "{font.face.2.normal.family.value}, Baskerville, Georgia, serif"
         }
+      },
+      "face3": {
+        "normal": {
+          "value": "{font.face.3.normal.family.value}, serif"
```

```suggestion
          "value": "{font.face.2.normal.family.value}, serif"
```
By having this reference "2", which is already defined, we don't need to define a "3" in the first file.

---

changes done